<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>attributeTreeViewDialog</class>
 <widget class="QWidget" name="attributeTreeViewDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>674</width>
    <height>744</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>674</width>
    <height>744</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/qtutils/fugue/table-import.png</normaloff>:/qtutils/fugue/table-import.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>These are the current attribute tree of the camera. Click &quot;Copy yo clipboard&quot; to copy the python dictionary representation of the current attributes.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="topMargin">
      <number>6</number>
     </property>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Visibility level</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="visibilityComboBox">
       <property name="editable">
        <bool>false</bool>
       </property>
       <property name="currentText">
        <string>Beginner</string>
       </property>
       <item>
        <property name="text">
         <string>Beginner</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/qtutils/fugue/table.png</normaloff>:/qtutils/fugue/table.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Expert</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/qtutils/fugue/tables.png</normaloff>:/qtutils/fugue/tables.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Guru</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/qtutils/fugue/tables-stacks.png</normaloff>:/qtutils/fugue/tables-stacks.png</iconset>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="topMargin">
      <number>8</number>
     </property>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeView" name="attributeTreeView"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="topMargin">
      <number>10</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="copyButton">
       <property name="text">
        <string>Copy to clipboard</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/qtutils/fugue/clipboard--arrow.png</normaloff>:/qtutils/fugue/clipboard--arrow.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>Close</string>
       </property>
       <property name="icon">
        <iconset theme="cancel">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../clones/qtutils/qtutils/icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
